### PR TITLE
Fix yiddishe-kop config

### PR DESCRIPTION
**Pull request by Yiddishe-Kop**
Added "language" facet, hopefuly fixing docsearch 🙏

I followed the [Nuxt config](https://github.com/algolia/docsearch-configs/blob/master/configs/nuxtjs_content.json) example.

### CODE REVIEW

**Comment by shortcuts** [comment]
Hi @Yiddishe-Kop,

It looks like you didn't merged the latest version of your config before submitting your PR, could you please resolve conflicts?

Thanks

**Comment by Yiddishe-Kop** [comment]
@shortcuts I've merged it.

File: configs/yiddishe-kop.json [hunk 1]
```diff
@@ -1,22 +1,35 @@
 {
   "index_name": "yiddishe-kop",
   "start_urls": [
-    "https://laravel-commerce.yiddishe-kop.com"
+    "https://laravel-commerce.yiddishe-kop.com",
+    "extra_attributes": {
+      "language": ["he"]
+    }
```
Comment (shortcuts) [inline configs/yiddishe-kop.json:7]:
```suggestion
    {
      "url": "https://laravel-commerce.yiddishe-kop.com",
      "extra_attributes": {
        "language": ["he"]
      }
    }
```

File: configs/yiddishe-kop.json [hunk 2]
```diff
@@ -1,22 +1,35 @@
...
-      "selector": "aside > div > ul > li.active > p",
+      "selector": "",
       "default_value": "Documentation"
     },
-    "lvl1": "div.flex div article.prose h1",
-    "lvl2": "div.flex div article.prose h2",
-    "lvl3": "div.flex div article.prose h3",
-    "lvl4": "div.flex div article.prose h4",
-    "text": "div.flex div article.prose p, div.flex div article.prose table, div.flex div article.prose li"
+    "lvl1": ".nuxt-content h1",
```
Comment (shortcuts) [inline configs/yiddishe-kop.json:15]: This selector doesn't exists anymore on your website
```suggestion
    "lvl1": "article h1",
```

Comment (Yiddishe-Kop) [reply configs/yiddishe-kop.json]: My bad, it got reverted by mistake. I fixed it.

**Pull request merged**